frontend/src/pages/team/Devices/dialogs/components/DeviceCredentialsDialog/OtcInstallSection.vue

```diff
@@ -88,6 +85,60 @@ export default {
         }
     },
     emits: ['select-os'],
+    computed: {
+        ...mapGetters('account', ['settings']),
+        commands () {
+            return {
+                script: {
+                    Windows: {
+                        title: 'Open Command Prompt or PowerShell as administrator and run:',
+                        command: 'powershell -c "irm https://raw.githubusercontent.com/FlowFuse/device-agent/refs/heads/main/installer/get.ps1|iex" && \\\n' +
```

Comment:
`&& \` causes error in `cmd`

```bash
'\' is not recognized as an internal or external command,
```

This should be on one line for windows `cmd` to work